app/controllers/users_controller.rb

```diff
@@ -0,0 +1,20 @@
+class UsersController < ApplicationController
+  
+  def new
+    @user = User.new
+  end
+
+  def create
+    @user = User.create(user_params)     
```

Comment:
Create saves already. You want to do `User.new`, then the next line handles the attempted save.